src/main/java/me/botsko/prism/players/PlayerIdentification.java

```diff
@@ -1,482 +1,360 @@
+        try (
+                Connection conn = Prism.getPrismDataSource().getConnection();
+                PreparedStatement s = conn.prepareStatement("INSERT INTO " + prefix
+                                + "players (player,player_uuid) VALUES (?,UNHEX(?))",
+                        Statement.RETURN_GENERATED_KEYS)
+                ) {
+            s.setString(1, player.getName());
+            s.setString(2, uuidToDbString(player.getUniqueId()));
+            s.executeUpdate();
+
+            ResultSet rs = s.getGeneratedKeys();
+            if (rs.next()) {
+                prismPlayer.setId(rs.getInt(1));
+                Prism.debug("Saved and loaded player " + player.getName() + " (" + player.getUniqueId()
+                        + ") into the cache.");
+                Prism.prismPlayers.put(player.getUniqueId(),
+                        new PrismPlayer(rs.getInt(1), player.getUniqueId(), player.getName()));
+            } else {
+                throw new SQLException("Insert statement failed - no generated key obtained.");
+            }
+            rs.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return prismPlayer;
+    }
+
+    /**
+     * Saves a fake player's name and generated UUID to the `prism_players` table.
+     * At this stage, we're pretty sure the UUID and username do not already exist.
+     *
+     * @param playerName String
+     * @return PrismPlayer
+     */
+    protected static PrismPlayer addPlayer(String playerName) {
+        PrismPlayer fakePlayer = new PrismPlayer(0, UUID.randomUUID(), playerName);
+        try (
+                Connection conn = Prism.getPrismDataSource().getConnection();
+                PreparedStatement s = conn.prepareStatement("INSERT INTO " + prefix
+                        + "players (player,player_uuid) VALUES (?,UNHEX(?))", Statement.RETURN_GENERATED_KEYS)
+                ) {
+            s.setString(1, fakePlayer.getName());
+            s.setString(2, uuidToDbString(fakePlayer.getUUID()));
+            s.executeUpdate();
+
+            ResultSet rs = s.getGeneratedKeys();
+            if (rs.next()) {
+                fakePlayer.setId(rs.getInt(1));
+                Prism.debug("Saved and loaded fake player " + fakePlayer.getName() + " into the cache.");
+                Prism.prismPlayers.put(fakePlayer.getUUID(), fakePlayer);
+            } else {
+                throw new SQLException("Insert statement failed - no generated key obtained.");
+            }
+            rs.close();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return fakePlayer;
+    }
+
+    /**
+     * Saves a player's UUID to the prism_players table. We cache the current
+     * username as well.
+     */
+    protected static void updatePlayer(PrismPlayer prismPlayer) {
+        prefix = Prism.config.getString("prism.mysql.prefix","prism_");
```

Comment:
Great addition of a default value of `prism_` if the config doesn't have `prism.mysql.prefix` defined.